ui/toolbox/esp32.xml

```diff
@@ -697,7 +697,9 @@
 
     <category name="OLED Display">
       <label text="SSD1306 OLED I2C Display"></label>
-      <button text="Install SSD1306 library" callbackKey="installPyLib"></button>
+      <button text="Install ssd1306 library" callbackKey="installPyLib"></button>
```

Comment:
We already have this library as `SSD1306`

Reply:
When it is installed with capital letters 'SSD' it doesn't seem to work. The lowercase version 'ssd' is seemingly the library everywhere else. I suggest lowercase could be used to be compatible with existing uses.